newrelic-integration-e2e/resources/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM golang:
+
+
+FROM newrelic/infrastructure:latest
```

Comment:
I am confused, why do we have two dockerfiles?

Reply:
now we have only one...:)  the `resources` folder was not required